subsys/bluetooth/controller/ll_sw/ull_adv_aux.c

```diff
@@ -171,6 +171,11 @@ uint8_t ll_adv_aux_ad_data_set(uint8_t handle, uint8_t op, uint8_t frag_pref, ui
 	if (!is_aux_new) {
 		sec_hdr_prev = *sec_hdr;
 	} else {
+		sec_pdu_prev->chan_sel = 0U;
```

Comment:
```
if (IS_ENABLED(CONFIG_BT_CTLR_CHAN_SEL_2)) {
		sec_pdu->chan_sel = sec_pdu_prev->chan_sel;
	} else {
		sec_pdu->chan_sel = 0U;
	}
```

Unrelated to this PR, but in Extended PDUs channel selection bit is RFU, since channel selection algorithm 2 must be supported when supporting Advertising Extensions.

Reply:
Removed the conditional compile, and now always setting to 0. In the following PR, I will address other such incorrect occurences. Thanks for pointing out.